tests/Serializer/ItemNormalizerTest.php

```diff
@@ -94,7 +95,7 @@ public function testNormalize()
         );
         $normalizer->setSerializer($serializerProphecy->reveal());
 
-        $this->assertEquals(['name' => 'hello'], $normalizer->normalize($dummy));
+        $this->assertEquals(['name' => 'hello'], $normalizer->normalize($dummy, null, ['resources' => []]));
```

Comment:
It passes without the arguments and without change, but I've modified it to test the new behavior.